src/topology/test/backpressure.rs

```diff
@@ -275,7 +275,7 @@ mod test_source {
             Ok(async move {
                 for i in 0.. {
                     let _result = cx.out.send(Event::from(format!("event-{}", i))).await;
-                    counter.fetch_add(1, Ordering::Relaxed);
+                    counter.fetch_add(1, Ordering::AcqRel);
```

Comment:
Just curious why this changed.

Reply:
I was trying to be more precise about the ordering when debugging why switching entirely to Tokio channels didn't seem to change the numbers as expected.